zcash_client_sqlite/src/prepared.rs

```diff
@@ -500,78 +579,6 @@ impl<'a, P> DataConnStmtCache<'a, P> {
             .map_err(SqliteClientError::from)
     }
 
-    /// Inserts a sent note into the wallet database.
-    ///
-    /// `output_index` is the index within the transaction that contains the recipient output:
-    ///
-    /// - If `to` is a Sapling address, this is an index into the Sapling outputs of the
-    ///   transaction.
-    /// - If `to` is a transparent address, this is an index into the transparent outputs of
-    ///   the transaction.
-    #[allow(clippy::too_many_arguments)]
-    pub(crate) fn stmt_insert_sent_note(
```

Comment:
Why were these moved? It makes review significantly harder (for me on my 13" laptop screen that can't show two copies of this PR side-by-side to check what is moved and what is altered without line wrapping).

Reply:
These were moved because they now require the `consensus::Parameters` constraint on `P` where previously they did not.